ArjunaJTS/integration/src/main/java/com/arjuna/ats/jbossatx/BaseTransactionManagerDelegate.java

```diff
@@ -55,6 +56,12 @@
 {
     private static final String LISTENER_MAP_KEY = "__TX_LISTENERS";
 
+    protected static final String LISTENERS_ENABLED_PROP = "com.arjuna.ats.jbossatx.BaseTransactionManagerDelegate.LISTENERS_ENABLED";
```

Comment:
@mmusgrov should not be the configuration props gathered under the `Bean`s? Should not be transferred under like `JTSEnvironmentBean` or similar?